resources/views/layouts/partials/header/autocomplete.blade.php

```diff
@@ -1,56 +1,76 @@
-@php $inputClasses = 'border !border-default !text-sm !min-h-0 outline-none !h-auto rounded !p-2 !bg-white w-full focus:!border-emphasis' @endphp
+@php $inputClasses = 'relative z-header-autocomplete border !font-sans !border-default !text-sm !min-h-0 outline-0 ring-0 !h-auto rounded-xl !pl-5 !pr-24 !py-3.5 !bg-white w-full focus:ring-transparent search-input' @endphp
 
-<label for="autocomplete-input" class="sr-only">@lang('Search')</label>
-<input
-    id="autocomplete-input"
-    placeholder="@lang('Search')"
-    class="{{ $inputClasses }}"
-    v-on:focus="$root.loadAutocomplete = true"
-    v-if="!$root.loadAutocomplete"
->
+<div v-if="!$root.loadAutocomplete" class="relative w-full">
+    <label for="autocomplete-input" class="sr-only">@lang('Search')</label>
+    <input
+        id="autocomplete-input"
+        placeholder="@lang('What are you looking for?')"
+        class="{{ $inputClasses }}"
+        v-on:focus="$root.loadAutocomplete = true"
+    >
+    <x-rapidez::autocomplete.magnifying-glass />
+</div>
 
 <autocomplete
-    v-if="$root.loadAutocomplete"
+    v-else
     v-on:mounted="() => window.document.getElementById('autocomplete-input').focus()"
     v-bind:additionals="{{ json_encode(config('rapidez.frontend.autocomplete.additionals')) }}"
     v-bind:debounce="{{ config('rapidez.frontend.autocomplete.debounce') }}"
     v-bind:size="{{ config('rapidez.frontend.autocomplete.size') }}"
-    class="w-full"
+    class="relative w-full"
     v-cloak
 >
-    <x-rapidez::reactive-base slot-scope="{ results, resultsCount, searchAdditionals, debounce, size, highlight }">
+    <x-rapidez::reactive-base slot-scope="{ results, resultsCount, searchAdditionals, overlay, showOverlay, debounce, size, highlight, searchLoading, startLoading, stopLoading }">
```

Comment:
Maybe better to use `slot-scope="autocomplete"` and use `autocomplete.overlay` instead of `overlay`, etc related to: https://github.com/rapidez/core/pull/601